sdk/servicebus/service-bus/tests.yml

```diff
@@ -14,6 +14,7 @@ jobs:
   - template: ../../../eng/pipelines/templates/jobs/archetype-sdk-integration.yml
     parameters:
       PackageName: "@azure/service-bus"
+      ResourceFileLocation: sdk/servicebus/test-resources.json
```

Comment:
We now use `ResourceServiceDirectory` instead of `ResourceFileLocation` 

This should be: 
```
ResourceServiceDirectory: servicebus
```